### PR TITLE
Travis-CI Fixes

### PR DESCRIPTION
**Pull request by pramsey**
PR for experimenting with fixes to the travis CI environment for ODBC-FDW

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,55 +31,35 @@ before_install:
...
   - sudo rm -rf /var/log/postgresql/
   - sudo rm -rf /etc/postgresql/
   - sudo apt-get -y remove --purge postgis-2.2
   - sudo apt-get -y autoremove
 
-  - sudo apt-get -y install postgresql-9.5=9.5.2-3cdb2
-  - sudo apt-get -y install postgresql-server-dev-9.5=9.5.2-3cdb2
-  - sudo apt-get -y install odbcinst
-  - sudo apt-get -y install odbc-postgresql
+  - sudo apt-get -y install postgresql-9.5
```
Comment (rafatower) [inline .travis.yml]: if we're using the official package then maybe there's no point in uninstalling and then re-installing it

File: Makefile [hunk 2]
```diff
@@ -28,7 +28,7 @@ DATA = odbc_fdw--0.0.1.sql \
 
 TEST_DIR = test/
 REGRESS = $(notdir $(basename $(sort $(wildcard $(TEST_DIR)/sql/*test.sql))))
-REGRESS_OPTS = --inputdir='$(TEST_DIR)' --outputdir='$(TEST_DIR)' --user='postgres' --load-extension=odbc_fdw
+REGRESS_OPTS = --inputdir='$(TEST_DIR)' --outputdir='$(TEST_DIR)' --user='postgres' --load-extension=odbc_fdw --debug
```
Comment (rafatower) [inline Makefile]: I think the `--debug` is no longer needed

**Comment by pramsey** [comment]
Very fabulous, @ethervoid, wow. @rafatower I'm going to take the learnings from here into the v10 branch and also take some CI fixes from other branches and get a little pg multi-version setup in place and then (finally) bring it all back to master.

File: .travis.yml (hunk 1 above)
Comment (ethervoid) [reply .travis.yml]: yep, you're right. Clearing.

File: Makefile (hunk 2 above)
Comment (ethervoid) [reply Makefile]: cleaning

**Comment by ethervoid** [comment]
Cleaned the PR

**Pull request merged**